### DISCUSSION TITLE
What is the maximum query string allowed in a vercel edge function?

### DISCUSSION THREAD

**Discussion by lifeiscontent** [vercel/community#5299] [Feature Question] [answered]
### Summary

from what I've seen, there is no documentation around how long a querystring can be in a vercel edge function

anyone know?

### Steps to Reproduce

TBD, but I could write an edge function that accepts a query string and just make it really long and test manually.

**Comment by leerob** [comment, marked as answer]
I believe the URL length limit is `14kb` on Vercel. Good flag we should likely have this in the docs 👍
